proposals/0393-parameter-packs.md

```diff
@@ -519,8 +520,6 @@ _ = tuplify(1, "hello", [Foo()]) // T := {Int, String, [Foo]}, value := {1, "hel
 
 Pack expansion expressions can also appear in an expression statement at the top level of a brace statement. In this case, the semantics are the same as scalar expression statements; the expression is evaluated for its side effect and the results discarded.
 
-Note that pack expansion expressions can also reference _type_ pack parameters, as metatypes.
```

Comment:
Does this deletion reflect a substantive change to what's proposed or is it an editorial change because the info isn't salient here?

Reply:
Just an editorial change! This sentence mentions one specific instance of types in the expression grammar, but type parameter packs can appear in pack expansion expression patterns in any context where a type is allowed in expressions. I believe this is already covered by this part of the text:

> A value parameter pack can only be referenced from a pack expansion expression. A pack expansion expression is written as `repeat expr`, where `expr` is an expression containing one or more value parameter packs or type parameter packs spelled with the `each` keyword.